lib/ui/window.dart

```diff
@@ -269,6 +269,24 @@ class Window {
   double get textScaleFactor => _textScaleFactor;
   double _textScaleFactor = 1.0;
 
+  /// The user setting instructing to always show time using the 24-hour format.
```

Comment:
I'd word this as "The setting indicating whether time should always be shown in the 24-hour format."

Reply:
Done.